packages/foreman/rubygem-sassc/rubygem-sassc.spec

```diff
@@ -66,44 +84,42 @@ mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
-
-mkdir -p %{buildroot}%{gem_extdir_mri}/%{gem_name}
-
-# hack to get the C extension where we expect it to be
-cp %{buildroot}/opt/theforeman/tfm/root/usr/share/gems/gems/sassc-%{version}/ext/libsass.so .%{gem_extdir_mri}/
-
-cp -a .%{gem_extdir_mri}/gem.build_complete %{buildroot}%{gem_extdir_mri}/
-cp -a .%{gem_extdir_mri}/*.so %{buildroot}%{gem_extdir_mri}/%{gem_name}/
-
-# Prevent dangling symlink in -debuginfo (rhbz#878863)
-# But this gem fails on it with error
-# File not found: /builddir/build/BUILDROOT/tfm-rubygem-sassc-2.2.1-1.el7.x86_64/opt/theforeman/tfm/root/usr/share/gems/gems/sassc-2.2.1/ext.
-# rm -rf %{buildroot}%{gem_instdir}/ext/
+%check
```

Comment:
given this package does not actually contain a native extension, we don't need the mkdir/copy dance and can use `ruby -I "%{buildroot}%{gem_libdir}" -e "require '%{gem_name}'"` here. do we want that? I'd say yes.

Reply:
I think that's good to verify the hack we've applied.

Reply:
You mean to try to "require 'sassc'"? Yeah, that definitely (even tho it does not fail on EL7 even when it can't find libsass.so - but does so on EL8 :woman_shrugging: )

I was more talking about the syntax of the check. Simple, or in line with the other packages?

Reply:
So I'll be a bit more verbose this time: I think it's good to very the gem actually works. If there's no native paths, there's no need to copy the directory I think.

Reply:
simplified, thanks